libksba will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libksba it will fail in the following way:

```
2024/11/12 23:12:11 INFO running step "test/pkgconf"
2024/11/12 23:12:11 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:11 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:11 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:11 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:11 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN + basename /home/build/melange-out/libksba-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN + dev_pkg=libksba-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN + apk info -L libksba-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN + grep -q ^Name: usr/lib/pkgconfig/ksba.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN + basename usr/lib/pkgconfig/ksba.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN + lib_name=ksba uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN + echo usr/lib/pkgconfig/ksba.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN + grep -q '^usr/lib/pkgconfig/ksba.pc$\|^usr/share/pkgconfig/ksba.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN + pkgconf --exists ksba uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN + pkgconf --print-errors --exists ksba uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN Package gpg-error was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN Perhaps you should add the directory containing `gpg-error.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:11 WARN Package 'gpg-error', required by 'ksba', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:12 INFO pod 3417c7d990b81df15048fae1abe3c2ffb157f1425d39eadffd3a85df02117650 terminated
2024/11/12 23:12:12 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:12 INFO   workspace dir: /home/user/tmp/melange-workspace-2901179373
2024/11/12 23:12:12 INFO   guest dir: /home/user/tmp/melange-guest-323264971
2024/11/12 23:12:12 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libksba] Error 1
```

--- a/libksba.yaml
+++ b/libksba.yaml
@@ -1,7 +1,7 @@
 package:
   name: libksba
   version: 1.6.7
-  epoch: 0
+  epoch: 1
   description: Libksba is a CMS and X.509 access library
   copyright:
     - license: GPL-2.0-or-later OR GPL-3.0-or-later
@@ -47,8 +47,12 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
+        - libgpg-error-dev
         - libksba
     description: libksba dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libksba-doc
     pipeline:

